test/mysql_handler/docker-compose.yml

```diff
@@ -30,9 +30,7 @@ services:
       MYSQL_ROOT_PASSWORD: securerootpass
 
   secretless-dev:
-    build:
-      context: ../..
-      dockerfile: Dockerfile.dev
+    image: secretless-dev
```

Comment:
is this really what we want here? in dev mode might we want to build using the current source if there are local changes since you last built the `secretless-dev` image?